x-pack/winlogbeat/module/security/_meta/docs.asciidoc

```diff
@@ -8,24 +8,82 @@ The security module processes event log records from the Security log.
 
 The module has transformations for the following event IDs:
 
+* 1100 - The event logging service has shut down.
+* 1102 - The audit log was cleared.
+* 1104 - The security log is now full.
+* 1105 - Event log automatic backup.
+* 1108 - The event logging service encountered an error while processing an incoming event published from %1
 * 4624 - An account was successfully logged on.
 * 4625 - An account failed to log on.
 * 4634 - An account was logged off.
 * 4647 - User initiated logoff (interactive logon types).
 * 4648 - A logon was attempted using explicit credentials.
 * 4672 - Special privileges assigned to new logon.
+* 4673 - A privileged service was called.
+* 4674 - An operation was attempted on a privileged object.
 * 4688 - A new process has been created.
 * 4689 - A process has exited.
+* 4697 - A service was installed in the system.
+* 4698 - A scheduled task was created.
+* 4699 - A scheduled task was deleted.
+* 4700 - A scheduled task was enabled.
+* 4701 - A scheduled task was disabled.
+* 4702 - A scheduled task was updated.
+* 4719 -  System audit policy was changed.
```

Comment:
very minor thing, but it would be nice if spacing was consistent. (this line has 2 spaces after "-", others have one space)

Reply:
Fixed